scripts/native_to_erc20/sendForeign.js

```diff
@@ -1,10 +1,10 @@
 const path = require('path')
 require('dotenv').config({
-  path: path.join(__dirname, '../.env')
+  path: path.join(__dirname, '../../.env')
 })
 const Web3Utils = require('web3-utils')
-const { web3Foreign } = require('../src/services/web3')
-const { sendTx, sendRawTx } = require('../src/tx/sendTx')
+const { web3Foreign } = require('../../src/services/web3')
+const { sendTx, sendRawTx } = require('../../src/tx/sendTx')
 
 const {
   USER_ADDRESS,
```

Comment:
Can ERC20_TOKEN_ADDRESS will be extracted from the bridge contract?

Reply:
Done.